frontend/src/components/customer/ProfileDetailsCard.tsx/ProfileDetailsCard.tsx

```diff
@@ -0,0 +1,480 @@
+                                                                },
+                                                            }}
+                                                            label="Country"
+                                                            {...register(
+                                                                "country",
+                                                                {
+                                                                    required:
+                                                                        "Country is required",
+                                                                }
+                                                            )}
+                                                            error={
+                                                                !!errors.country
+                                                            }
+                                                            helperText={
+                                                                errors.country
+                                                                    ?.message
+                                                            }
+                                                            FormHelperTextProps={{
+                                                                sx: {
+                                                                    fontSize:
+                                                                        "1rem",
+                                                                },
+                                                            }}
+                                                        />
+                                                    )}
+                                                />
+                                            </Grid>
+                                        </Grid>
+                                    </Grid>
+                                </Grid>
+                            </Grid>
+                        </Grid>
+                        <Grid item sx={{ mt: "1.5em" }}>
+                            <Divider />
+                        </Grid>
+                        <Grid item>
+                            <Grid
+                                container
+                                direction="row"
+                                justifyContent="flex-end"
+                                spacing={3}
+                                sx={{ mt: "1.5rem", mb: "1.5rem" }}
+                            >
+                                <Grid item>
+                                    <Button
+                                        type="button"
+                                        variant="contained"
+                                        onClick={handleCancel}
+                                        sx={{
```

Comment:
Check the `Button` api provided by MUI, there should be a variant that can help with the cancel type of button so you need not to declare the bg color and hover effects. (Currently, hovering it makes it turn purple...)